docs/README.md

```diff
@@ -27,26 +27,26 @@ air
 
 ### Getting Started
 
-- [Building](building.html)
-- [Running Examples](running.html)
-- [AIRCC](aircc.html)
+- [Building](building.md)
+- [Running Examples](running.md)
+- [AIRCC](aircc.md)
 
 ### Generated MLIR Documentation
-- [AIR Dialect](AIRDialect.html)
-- [AIRRt Dialect](AIRRtDialect.html)
-- [AIR Transform Passes](AIRTransformPasses.html)
-- [AIR Conversion Passes](AIRConversionPasses.html)
-- [Transform Dialect Extensions](AIRTransformOps.html)
+- [AIR Dialect]
+- [AIRRt Dialect]
+- [AIR Transform Passes]
+- [AIR Conversion Passes]
+- [Transform Dialect Extensions]
```

Comment:
Revert this

Reply:
The platform docs links will be broken after navigating to https://github.com/Xilinx/mlir-air/blob/main/docs/building.md directly, but when I set it up I thought it was nicer to have the platform specific docs be the `README.md` when navigating to `https://github.com/Xilinx/mlir-air/tree/main/platforms/<platform>`.  We could instead move the platform `README.md`s to the docs folder. Or maybe there are other suggestions?